commands/slash/seek.js

```diff
@@ -67,7 +72,7 @@ const command = new SlashCommand()
 					new MessageEmbed()
 						.setColor(client.config.embedColor)
 						.setDescription(
-							`Cannot seek current playing track. This may happened because seek duration has exceeded track duration`,
+							`Cannot seek current playing track. This may happened because seek duration has exceeded track duration or a incorrect time format`,
```

Comment:
`Unable to seek current playing track. This may be due to exceeding track duration or incorrect time format. Please check and try again`